action.yml

```
@@ -199,7 +247,7 @@ runs:
         TERRAFORM_OUTPUT_FILE="./terraform-${GITHUB_RUN_ID}-output.txt"
 
         tfcmt \
-        --config ${GITHUB_ACTION_PATH}/config/summary.yaml \
+        --config ${{ github.action_path }}/config/summary.yaml \
```

Rollback because of this PR https://github.com/cloudposse/github-action-atmos-terraform-plan/pull/91
```suggestion
        --config ${GITHUB_ACTION_PATH}/config/summary.yaml \
```